source/ore/routes/list-users.rst

```diff
@@ -0,0 +1,83 @@
+==========
+List Users
+==========
+
+**GET /api/users**
+
+Returns a list of users based on given criteria.
```

Comment:
It let's people interact with projects in their application and list information about the author/developers. OrePlugin uses it, for instance.